### DISCUSSION TITLE
Add backup notification

### DISCUSSION THREAD

**Discussion by SerhiiZahuba** [DanSheps/netbox-config-backup#23] [Ideas]
Please consider adding a notification about successful and unsuccessful backups. For example email + telegram. Thanks

**Comment by DanSheps** [comment]
Can you open a FR for this?

**Comment by SerhiiZahuba** [comment]
What do you mean FR

**Reply by DanSheps** [reply]
Feature Request

**Comment by SerhiiZahuba** [comment]
https://github.com/DanSheps/netbox-config-backup/issues/27
